controllers/Login.py

```diff
@@ -0,0 +1,49 @@
+import bcrypt
+from flask import Flask,request,jsonify,Blueprint
+from Models.Users import Users
+
+
+app_login = Blueprint('app_login',__name__)
+
+@app_login.route("/login",methods=["POST"])
+def login():
+
+    response = {}
+
+    try:
+        #username_ = request.form['username']
+        #password_ = request.form['password']
+
+        username_ = request.json.get('username', None)
+        password_ = request.json.get('password', None)
```

Comment:
Have a look at the marshmallow package in controllers/MatchUsers.py. It already checks for missing request params in an elegant way!